docs/AzureGPU.md

```diff
@@ -20,10 +20,18 @@ Example: ```az group create -l eastus -n GPU-Folding-COVID19```
 
 * Now that you have a new RG created, you can go ahead and deploy the GPU VM in a single command: <br>
 
-```az deployment group create --name "folding" --resource-group "GPU-Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json```
+```az deployment group create --name "folding" --resource-group "GPU-Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json --parameters https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.parameters.json```
 
 **Note: Don't forget to match the resource group name based on the one you created**
 
+# Advanced VM Deployment
+
+The ARM Template leverages a parameters file with preconfigured values for things like username and password.   While you can clone this repo and make the necessary modifications to the files to override the defaults, there is a quicker path leveraging the same ```--parameters``` cli argument.  For example, the following command would do the same deployment with a customized username and password:
+
+```az deployment group create --name "folding" --resource-group "Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json --parameters adminUsername="<some username>" adminPasswordOrKey="<some strong password>" ```
+
+**Note:  You'll need to replace ```<some username>``` and ```<some strong password>``` with your own values.**
+
 ![](../img/Azure/az_01.png)
 
 ![](../img/Azure/az_02.png)
```

Comment:
Fix the "Reboot & Shutdown" section at the end of the doc. Currently, it is aligned with Windows.